target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,13 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_usbdev "phone" "Phone" "${board}:white:phone" "1"
+	ucidef_set_led_usbdev "usb" "USB" "${board}:white:usb" "1"
```

Comment:
use ucidef_set_led_usbport here. This way multiple usb ports can control one LED.

Something like:

```
ucidef_set_led_usbport "usb" "USB" "${board}:white:usb" "usb1-port1" "usb2-port1" "usb3-port1" "usb4-port1"
```

should work. Find a more detailed explaination at https://github.com/lede-project/source/pull/414/commits/db9904cb8bb38c2cd2a856e0ec7d83ebe0528d5a#discussion_r88208884